docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -530,8 +501,8 @@ persistence property *wildfly.jpa.hibernate.search.module* to either
 any other module identifier to inject a different module. +
 For example you could set
 *wildfly.jpa.hibernate.search.module=org.hibernate.search.orm:5.4.0.Alpha1*
-to use the experimental version 5.4.0.Alpha1 instead of the provided
-module; in this case you'll have to download and add the custom modules
+to use the version 5.4.0.Alpha1 instead of the provided
```

Comment:
The version of Search referenced here should probably be updated too. Something like `5.11.0.Alpha1` (it doesn't exist but it's a better example than 5.4 as it wouldn't work).

That being said, we don't provide the modules anymore but feature packs.

Reply:
Updating to 5.11.0.Alpha1...

Reply:
Regarding feature packs versus modules, let me know what we should change to say.